app/CMakeLists.txt

```diff
@@ -57,6 +59,11 @@ endif()
 
 target_link_libraries(${PROJECT_NAME} PRIVATE vkb__core vkb__filesystem apps plugins)
 
+# Force multi-config generators (VS, Xcode, Ninja Multi-Config) to build for the Release config when cmake build type is Release
+if(${CMAKE_BUILD_TYPE} STREQUAL "Release")
+	set(CMAKE_CONFIGURATION_TYPES "Release;RelWithDebInfo" CACHE STRING "Release build types for multi-config generators" FORCE)
```

Comment:
This feels a little heavy handed.  What are we fixing here?

Reply:
This forces VS and Xcode to build for release when the CMake build type is release.  If the CMake build tyoe is Debug, VS and Xcode will build for Debug.  This is a usability and convenience thing.

Reply:
I understand, I think my hesitation is that it will wipe out the cache and forces the update everytime there is a change to CMakeLists or to a project.  Lemme do some thinking, there might be a way to achieve this with a less heavy handed change.

Reply:
Again, this has now been reverted.